openapi.yaml

```
@@ -0,0 +1,1791 @@
+      description: Metadata of a dispatchable call from a FRAME pallet.
+    PalletStorage:
+      type: object
+      properties:
+        prefix:
+          type: string
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletStorageItem'
+    Pallet:
+      type: object
+      properties:
+        index:
+          type: string
+          format: unsignedInteger
+        name:
+          type: string
+        storage:
+          $ref: '#/components/schemas/PalletStorage'
+        calls:
+          type: array
+          items:
+            $ref: '#/components/schemas/Dispatchable'
+        events:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletEvent'
+        constants:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletConstant'
+        errors:
+          type: array
+          items:
+            $ref: '#/components/schemas/PalletError'
+      description: Metadata of a FRAME pallet.
+    RuntimeDispatchInfo:
+      type: object
+      properties:
+        weight:
+          type: string
+          description: Extrinsic weight.
+        class:
+          type: string
+          description: Extrinsic class.
+          enum:
+          - Normal
+          - Operational
+          - Mandatory
+        partialFee:
+          type: string
+          description: Transaction fee, not including any tip.
+          format: unsignedInteger
+      description: RuntimeDispatchInfo for the transaction. Includes the `partialFee`.
+    Extrinsic:
+      type: object
+      properties:
+        method:
+          type: string
+          description: Extrinsic method, `{module}.{function}`.
+        signature:
+          $ref: '#/components/schemas/Signature'
+        nonce:
+          type: string
+          description: Account nonce, if applicable.
+          format: unsignedInteger
+        args:
+          type: object
+          description: Object of arguments keyed by parameter name.
+        tip:
+          type: string
+          description: Any tip added to the transaction.
+          format: unsignedInteger
+        hash:
+          type: string
+          description: The transaction's hash.
+          format: hex
+        info:
+          $ref: '#/components/schemas/RuntimeDispatchInfo'
+        events:
+          type: array
+          description: An array of `SanitizedEvent`s that occurred during extrinsic
+            execution.
+          items:
+            $ref: '#/components/schemas/SanitizedEvent'
+        success:
+          type: boolean
+          description: Whether or not the extrinsic succeeded.
+        paysFee:
+          type: boolean
+          description: Whether the extrinsic requires a fee. Careful! This field relates
+            to whether or not the extrinsic requires a fee if called as a transaction.
+            Block authors could insert  the extrinsic as an inherent in the block
```

I thought we made this PaysFee do this check? This is only true when the extrinsic is signed and `DispatchInfo::Pays` is true.
```suggestion
            Block authors could insert the extrinsic as an inherent in the block
```